src/ralph/admin/templates/admin/change_form.html

```diff
@@ -83,22 +84,3 @@
     </form>
   </div>
 {% endblock %}
-{% block admin_change_form_document_ready %}
```

Comment:
check this out: https://github.com/allegro/ralph/pull/1671/files#diff-633f75b2b91e30c5818083aab400d14e

One of us (after merging first PR) will have to tweak it's changes